src/main/java/org/example/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -66,4 +66,11 @@ public List<BookDto> search(BookSearchParameters params) {
                 .map(bookMapper::toDto)
                 .toList();
     }
+
```

Comment:
Should we set categories during saving and updating book?

Reply:
Why not?

Reply:
Check please and text here, if during saving and updating book categories also set

Reply:
Sorry, missunderstanding